libs/barista-components/sunburst-chart/src/sunburst-chart.spec.ts

```diff
@@ -162,14 +151,14 @@ describe('DtSunburst', () => {
       it('should be ABSOLUTE by default', () => {
         fixture.detectChanges();
 
-        expect(component.labelAsAbsolute).toBeTruthy();
+        expect(component._labelAsAbsolute).toBeTruthy();
```

Comment:
Avoid accessing internals in tests as well, you should always test the end result here and not the property having the correct values. If the check in the template is wrong the property might have the correct value but you might show the exact opposite of what you like. 
So you could check whether the text inside the chart is formatted correctly and that the label of the selected slice is also formatted

Reply:
I think this is still unresolved :)